homeassistant/components/recorder/purge.py

```diff
@@ -125,6 +125,9 @@ def purge_old_data(
         _purge_old_recorder_runs(instance, session, purge_before)
     if repack:
         repack_database(instance)
+    with session_scope(session=instance.get_session()) as session:
```

Comment:
Do we need a new session for updating the managers, or can we reuse the session used when purging?

Reply:
Fixed in cb1682a